CONTRIBUTING.md

```diff
@@ -101,54 +92,52 @@ If you prefer to setup your own environment instead, make sure you have these pr
 
 To build all packages:
 
-4. Install and link all dependencies (`rush update`)
-5. Build the code base (`rush rebuild`)
+4. Install and link all dependencies (`pnpm install`)
+5. Build the code base (`pnpm build`)
```

Comment:
Running `pnpm build` under root folder would build all projects but in specific folder would only build that project?

Reply:
In the root, pnpm build will build all projects, however building it in a folder will only build that one item and its referenced projects.